### DISCUSSION TITLE
Extending Cake.Yarn with publish command

### DISCUSSION THREAD

**Discussion by gjhommersom** [cake-build/cake#3842] [Extension requests]
A long time ago I made a pull request on the Cake.Yarn repository: https://github.com/cake-contrib/cake-yarn/pulls.

This pull request was never handled and I never chased it. Mostly because my attention shifted elsewhere. I'm using it again and prefer to add to this central repository instead of forking it. This way I don't have to handle the build and publish process and everybody gets to enjoy the changes.

Can this pull request be merged?
And how will additional changes be handled?

**Comment by gep13** [comment]
@gjhommersom thanks for reaching out!

If there hasn't been any traction from the maintainers on that repository, we can look to escalate, by either taking on the work within the Core Cake Team, or perhaps adding you as a maintainer on the repository, if this is something that you are interested in.  Forking the project, and duplicating the addin, is something that we would like to avoid if possible.

Have you tried to reach the maintainer in any other ways, or was it just through the issue/PR?
